### PR TITLE
[FR] input-output.rst translation

### PR DESCRIPTION
**Pull request by Moutard3**
Thanks @Mydexios for helping me in this translation!

### CODE REVIEW

File: fr/console-and-shells/input-output.rst [hunk 1]
```diff
@@ -1,37 +1,39 @@
...
 
-While CakePHP comes with a few command helpers you can create more in your
-application or plugins. As an example, we'll create a simple helper to generate
-fancy headings. First create the **src/Command/Helper/HeadingHelper.php** and put
-the following in it::
+Alors que CakePHP est fourni avec quelques helpers de commande, vous pouvez
+en créer d'autres dans votre application ou vos plugins. À titre d'exemple,
+nous allons créer un helper simple pour générer des titre élégants.
+Créez d'abord le fichier **src/Command/Helper/HeadingHelper.php** et mettez
+ce qui suit dedans : ::
```
Comment (markstory) [inline fr/console-and-shells/input-output.rst]: You don't need 3 `:` only 2 are required.

File: fr/console-and-shells/input-output.rst [hunk 2]
```diff
@@ -48,29 +50,31 @@ the following in it::
         }
     }
 
-We can then use this new helper in one of our shell commands by calling it::
-
-    // With ### on either side
+Nous pouvons alors utiliser ce nouvel Helper dans l'une de nos commandes
+shell en l'appelant : ::
```
Comment (markstory) [inline fr/console-and-shells/input-output.rst]: You don't need 3 `:` only 2 are required.

File: fr/console-and-shells/input-output.rst [hunk 3]
```diff
@@ -48,29 +50,31 @@ the following in it::
...
 
-Table Helper
-------------
+L'Helper Table
+--------------
 
-The TableHelper assists in making well formatted ASCII art tables. Using it is
-pretty simple::
+Le TableHelper aide à faire des tableaux d'art ASCII bien formatés.
+L'utiliser est assez simple : ::
```
Comment (markstory) [inline fr/console-and-shells/input-output.rst]: You don't need 3 `:` only 2 are required.

File: fr/console-and-shells/input-output.rst (hunk 1 above)
Comment (Moutard3) [reply fr/console-and-shells/input-output.rst]: But when using only 2 : (with the space before) no : is redered at all

Comment (markstory) [reply fr/console-and-shells/input-output.rst]: Odd. Sphinx usually inserts a `:` when `::` is used. For example https://book.cakephp.org/3.0/en/core-libraries/events.html and https://raw.githubusercontent.com/cakephp/docs/3.0/en/core-libraries/events.rst

Comment (Moutard3) [reply fr/console-and-shells/input-output.rst]: sublime_text_2018-08-02_19-39-59
As you can see, putting a space before the `::` doesn't automatically add the `:`

**Comment by dereuromark** [comment]
@antograssiot Then you can merge.

**Pull request merged**